packages/gatsby-plugin-utils/src/test-plugin-options-schema.ts

```diff
@@ -0,0 +1,38 @@
+import { Joi } from "./joi"
+import { GatsbyNode } from "gatsby"
+
+interface ITestPluginOptionsSchemaReturnType {
+  errors: Array<string>
+  isValid: boolean
+}
+
+export function testPluginOptionsSchema<PluginOptions = object>(
```

Comment:
```suggestion
export function testPluginOptionsSchema<PluginOptions = Record<string, unknown>>(
```

Reply:
I'm actually not really sure of the value of making this generic. We're not using the type anywhere. I'd remove it, and instead type the prop as `Record<string, unknown>`